projects/react-boilerplate-ts/app/containers/LanguageProvider/reducer.ts

```diff
@@ -4,22 +4,25 @@
  *
  */
 
-import { fromJS } from 'immutable';
-
 import { CHANGE_LOCALE } from './constants';
 import { DEFAULT_LOCALE } from '../../i18n';
+import { ChangeLocaleAction } from './actions';
+import { AppAction } from 'reducers';
```

Comment:
Confused by how this works, it looks like elsewhere relative imports are used, but VSCodes auto-import is doing it this way. Looks like it's a package called "reducers" rather than a file in the root of the project.

Reply:
In tsconfig there are some settings that relate to this, one is "moduleResolution", which can be "classic" or "node", we use "node" which is the modern approach and relies mostly on relative paths, however there is also "baseUrl", which specifies the the base resolution starting point for non-relative module names. Ours is set to 'app' thus all absolute names will resolve from here, including reducers.ts.

Reply:
Ah makes sense. What do you think about changing the root to be a level up. Then ‘import { AppAction } from “app/reducers”’ for clarity? Or will that break loads of stuff?